controllers/tradingController.js

```diff
@@ -0,0 +1,30 @@
+const tradeModel = require('../models/trading')
+/**
+ * New Trading Request
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const trade = async (req, res) => {
+  try {
+    const { username } = req.params
+    if (!username) {
+      return res.boom.forbidden('Invalid username')
+    }
+
+    const { canUserTrade, errorMessage, userBalance } = await tradeModel.trade(req.body, username)
+
+    if (!canUserTrade && canUserTrade !== undefined) {
+      return res.boom.forbidden(errorMessage)
+    }
+
+    return res.status(200).json({ userBalance })
```

Comment:
.status(200) is not required as it is by default.

Reply:
https://github.com/Real-Dev-Squad/website-backend/pull/222/commits/c47c5f50b9d54d8fd2e4436aace96f4660f4bf9c